src/itemChoice/index.tsx

```diff
@@ -1,135 +1,118 @@
-import React, { PureComponent } from 'react'
+import React, { PureComponent, Fragment } from 'react'
 import cc from 'classcat'
-import isEmpty from 'lodash.isempty'
 
-import prefix from '_utils'
-import ChevronIcon from 'icon/chevronIcon'
 import Loader from 'loader'
 import style from './style'
 
+import Item from '_utils/item'
+import ChevronIcon from 'icon/chevronIcon'
+
 export enum ItemChoiceStatus {
   DEFAULT = 'default',
   LOADING = 'loading',
   CHECKED = 'checked',
 }
 
-interface TypeProps {
-  readonly className?: Classcat.Class,
-  readonly href?: string,
-  readonly onClick?: (event: React.MouseEvent<HTMLElement>) => void,
-  readonly onBlur?: (event: React.FocusEventHandler<HTMLElement>) => void,
-  readonly onFocus?: (event: React.FocusEventHandler<HTMLElement>) => void,
-  readonly onMouseDown?: (event: React.MouseEvent<HTMLElement>) => void,
-}
-
 export interface ItemChoiceProps {
-  readonly className?: Classcat.Class,
-  readonly href?: string | JSX.Element,
-  readonly label?: string,
-  readonly subLabel?: string,
-  readonly children?: React.ReactNode,
-  readonly leftAddon?: React.ReactNode,
-  readonly rightAddon?: React.ReactNode,
-  readonly highlighted?: boolean,
-  readonly selected?: boolean,
-  readonly status?: ItemChoiceStatus,
-  readonly onDoneAnimationEnd?: () => void,
-  readonly onClick?: (event: React.MouseEvent<HTMLElement>) => void,
-  readonly onBlur?: (event: React.FocusEventHandler<HTMLElement>) => void,
-  readonly onFocus?: (event: React.FocusEventHandler<HTMLElement>) => void,
-  readonly onMouseDown?: (event: React.MouseEvent<HTMLElement>) => void,
+  readonly className?: Classcat.Class
+  readonly href: string | JSX.Element
+  readonly label?: string
+  readonly subLabel?: string
+  readonly highlighted?: boolean
```

Comment:
It seems to be removed from `AutocompleteListItem`, did you forget to remove it here ?

Reply:
Or it's a breaking change and the meaning changed ?